### PR TITLE
add spl governance v2 support

### PR DESCRIPTION
**Pull request by GregoryNEUT**

### CODE REVIEW

**Review by acamill — APPROVED** [review]
LGTM proper!

File: programs/anchor-comp/src/spl_governance_v2.rs [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+    let beneficiary = get_pubkey_from_opt_account_info(remaining_accounts_iter.next());
+
+    let ix = spl_governance::instruction::relinquish_vote(
+        &spl_governance_program_id::ID,
+        ctx.accounts.governance.key,
+        ctx.accounts.proposal.key,
+        ctx.accounts.voter_token_owner_record.key,
+        ctx.accounts.governing_token_mint.key,
+        governance_authority,
+        beneficiary,
```
[the remark this answers is not in the document]
Comment (GregoryNEUT) [reply programs/anchor-comp/src/spl_governance_v2.rs:116]: Idk for now, just made the wrapping :) 

Now I'm digging to see the infos to provide or not on each proposals

**Comment by GregoryNEUT** [comment]
Still good after looking at the last changes boss? 🙏

**Review by acamill — APPROVED** [review]
Lgtm!

**Pull request merged**